src/msw/graphicsd2d.cpp

```diff
@@ -360,6 +360,221 @@ HRESULT WINAPI wxD3D11CreateDevice(
 }
 #endif
 
+#if wxUSE_PRIVATE_FONTS
+
+// This function is defined in src/msw/font.cpp.
+extern const wxArrayString& wxGetPrivateFontFileNames();
+
+namespace
+{
+    IDWriteFontCollection* gs_pPrivateFontCollection = NULL;
+
+    typedef unsigned int wxDirect2DFontKey;
+
+    class wxDirect2DFontFileEnumerator : public IDWriteFontFileEnumerator
+    {
+    public:
+        wxDirect2DFontFileEnumerator(IDWriteFactory* pFactory)
```

Comment:
```suggestion
        explicit wxDirect2DFontFileEnumerator(IDWriteFactory* pFactory)
```
Just to avoid any strange conversions.